src/components-work/AuthenticationTestimonials/AuthenticationTestimonials.tsx

```diff
@@ -18,7 +18,7 @@ const items = [
   },
 ];
 
-class LoginTestimonials extends React.Component {
+class AuthenticationTestimonials extends React.Component {
```

Comment:
we should just make this `GuestTestimonials` or `HostTestimonials`; AuthTestimonials doesn't make sense?

Reply:
wasnt sure if we were actually going to use this testimonial carousel, wiull rename to `UserTestimonials`